packages/verification-vue-component/src/components/Verification/items/cards/VerificationCard.vue

```diff
@@ -80,7 +80,7 @@
                     <div class="verifier-name">
                         <span class="label">{{ _$t('verification.result.meta.issuerVerifiedBy') }}</span>
                         <div v-if="verificationItem.issuerVerifiedImg" class="verifier-image">
-                            <img :src="verificationItem.issuerVerifiedImg" alt=""/>
+                            <img :src="(verificationItem.issuerVerifiedImg).split('.png')[0] + '_redesign.png'" alt=""/>
```

Comment:
It's quite limited and works only for .png, for the moment enough, but I would add a comment why we did that and that it should be replaced later, when we have the full event structure.

Reply:
Added a comment